pkg/kubernetes/podhelper/containers.go

```diff
@@ -31,19 +33,14 @@ func (check EnvoySidecarImageCheck) Info() string {
 
 // Run implements common.Runnable
 func (check EnvoySidecarImageCheck) Run() error {
-	cfg := kuberneteshelper.GetOsmConfigurator(check.pod)
-
 	for _, container := range check.pod.Spec.Containers {
-		if container.Image == cfg.GetEnvoyImage() {
+		if container.Image == check.cfg.GetEnvoyImage() {
```

Comment:
If you decide to print the info in https://github.com/openservicemesh/osm-health/pull/57/files#r686106973 , let's also print the same info here.